### ISSUE TITLE
GPU 4 card learning speed question.

### ISSUE THREAD

**Issue by donghun9808** [tensorflow/models#3768]
tensorflow "Object Detection" API is used.

The model is "faster_rcnn_resnet101_coco_2017_11_08".

CPU : intel XEON E5-2620 v4 * 2
GPU : ZOTAC NVIDIA GeForce GTX 1080 Ti * 4
RAM : DDR4-2400(1200 MHz) 16Gb * 8 (128Gb)

OS : windows10
CUDA : cuda_9.0.176_win10
cudnn : cudnn-9.0-windows10-x64-v7

images size : 3840*2748
data set : 1880 (train set : 1234, test set 640)

gpu  4 cards >> batch_size : 4 in faster_rcnn_resnet101_coco.config
num_clones : 4 in [train.py](https://github.com/tensorflow/models/blob/master/research/deeplab/train.py#L38)

When you start learning, you will get 90 ~ 100% of cpu, 30Gb of RAM, 14% of compute_0 of GPU and the learning time is about 2 seconds per step.

Is there a reason why the cpu goes up to 90% and the gpu does not go up?

Is there any reason why ram can go up to 30Gb and not use more than 120Gb?

Are there CPU, RAM, and GPU usage needs to be fixed in the code?

Learning is too slow.
I ask for help.
Thank you.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
